srcpkgs/rbw/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'rbw'
+pkgname=rbw
+version=1.3.0
+revision=1
+build_style=cargo
+depends="pinentry"
+short_desc="Unofficial command line client for Bitwarden"
+maintainer="Omar Zeghouani <omarzeghouanii@gmail.com>"
+license="MIT"
+homepage="https://git.tozt.net/rbw"
+distfiles="${homepage}/snapshot/${pkgname}-${version}.tar.gz"
+checksum=7255e322b8c6bd6ee9da00f4c54211e94a132abec15b9f51b8351af31a4744ac
+
+do_build() {
+	cargo build --release --locked
+
+	cargo run --release --locked --bin rbw -- gen-completions bash > rbw.bash
+	cargo run --release --locked --bin rbw -- gen-completions fish > rbw.fish
+	cargo run --release --locked --bin rbw -- gen-completions zsh  > rbw.zsh
```

Comment:
This is probably the problem with cross compilation (arm*, aarch64*). `cargo run` compiles for the target architecture (arm) and then tries to run the arm binary on the host computer (x86_64) which fails.